astro-i18next.config.mjs

```diff
@@ -0,0 +1,13 @@
+/** @type {import('astro-i18next').AstroI18nextConfig} */
+export default {
+  defaultLocale: "de",
+  locales: ["de", "en"],
+
+  // workaround for for leading '/' in path
+  // will be fixed in next release
+  i18nextServer: {
+    backend: {
+      loadPath: "./public/locales/{{lng}}/{{ns}}.json"
+    }
+  }

```

Comment:
Can you elaborate on why you added this? I am unsure if I understand it correctly from the docs, but it sounds like it loads localization on the backend side of the project, for example, in NextJS. Since Astro does not have such a thing, it won't do anything.

Reply:
This is just a workaround because astro-i18next uses i18next and it's configurations. There is currently a bug left, which causes locale paths to have a leading '/'. That results in a FileNotFoundError. Through the loadPath attribute you can configure how i18next and therefore astro-i18next loads the translation files. astro-i18next uses the server configuration by default.

Reply:
It is also the suggested workaround by the author of this plugin:

> Hey,
> I'll fix it in next release, in the meantime you can override the i18next config to set your locales path.
> There's an issue referring to this on github: https://github.com/yassinedoghri/astro-i18next/issues/79
> 
> Hope it helps,
> Yassine.

Reply:
Thanks for the explanation, that makes sense! Can you maybe link the issue (https://github.com/yassinedoghri/astro-i18next/issues/79) in the comment so that it's easier to find the cause?

Reply:
Sure